dotCMS/src/main/java/com/dotmarketing/portlets/structure/action/EditFieldAction.java

```diff
@@ -552,4 +590,15 @@ public String hostNameUtil(ActionRequest req) {
         return h.getTitle()!=null?h.getTitle():"default";
     }
 
+    /**
+     * Validate if the string is one of the accepted Db boolean values
+     * @param value The boolean string to check
+     * @return true if is a valid boolean, false if not
+     */
+    private boolean isDBBoolean(final String value){
+        return (value.toLowerCase().trim().equals("t") || value.toLowerCase().trim().equals("true")
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the number of conditional operators (5) used in the expression (maximum allowed 4). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1067)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "t" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1132)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "true" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1132)